src/components/UnorderedList/UnorderedList.stories.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
Instead of this default comment, could you use something like this:
```suggestion
/**
 * @file Unordered list stories.
 * @copyright IBM Security 2019
 */
```